Implementation_in_Python/CICIDS2017_Security_Data/Sample_Results_Security_Example/README.md

```diff
@@ -0,0 +1,34 @@
+## Wednesday Attack
+<p align="justify">This attack occurred on Wednesday, July 5, 2017, and different types of attacks have been recorded such as DoS / DDoS, DoS slowloris (9:47 – 10:10 a.m.), DoS Slowhttptest (10:14 – 10:35 a.m.), DoS Hulk (10:43 – 11 a.m.), and DoS GoldenEye (11:10 – 11:23 a.m.). </p>
+
+<p align="center">
+ <img src="https://github.com/ISorokos/SafeML/blob/master/Implementation_in_MATLAB/Sample_Results_Security_Example/Security_Dataset_W01.png" alt="Wednessday_Attack">
+ <figcaption>Figure 1. Confusion matrix for Wednesday Security Intrusion Detection in CICIDS2017 Dataset</figcaption>
+</p>
+<p align="justify">Figure 2 shows a sample result of six statistical measures (one pdf-based and five cdf-based) vs. accuracy of the classifier. In this example, traditional classifiers including 'Naive Bayes','Discriminant Analysis','Classification Tree', and 'Nearest Neighbor' have been used. </p>
+<p align="center">
+ <img src="https://github.com/ISorokos/SafeML/blob/master/Implementation_in_MATLAB/Sample_Results_Security_Example/Security_Dataset_W02v2.png" alt="Wednessday_Attack">
+ <figcaption>Figure 2. Sample statistical distance measures vs. accuracy</figcaption>
+</p>
+<p align="justify">The above figure has been generated over 100 times, and each time 70% of data has been randomly extracted for testing and training and the rest has been used for accuracy estimation. If we connect red lines of each box plot we can understand how each measure can react to the accuracy change on average. In addition, this plot can show which method has less variation.</p>
```

Comment:
@WillBridges1999 "the above figure has been generated over 100 times", do you mean the data/experiments were executed over (more than?) 100 iterations ?

Reply:
Sorry, I think that's just the copied README from the MATLAB code implementation. I should probably have taken that out but it's up to you :) That part you've mentioned seems like a mistake for sure. Good spot!